src/ert/scheduler/openpbs_driver.py

```diff
@@ -244,6 +244,7 @@ async def poll(self) -> None:
                     "qstat",
                     "-x",
                     "-w",  # wide format
+                    "-E",  # group job ids
```

Comment:
Can the comment be made more precise, check the manual, I think it is more like group by PBS server. Maybe also comment that it is critical for speed.

Reply:
(at least the reason for why -E is added should be either in a comment or in the commit message)